Sources/ScipioKit/SwiftPM+Compatibility.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import TSCBasic
+import Basics
+import PackageModel
+
+// TODO: Remove TSCBasics
+// Since Swift 5.10, SwiftPM removes swift-tools-support-core(TSC)
+// So all interfaces are replaced from TSCBasics.AbsolutePath to Basics.AbsolutePath
+// These are almost identical. Unfortunately, Scipio still uses TSC versions of them.
+// It's better to remove TSC dependencies from Scipio.
+// However we just provides utils to bridge them at this time
+#if swift(>=5.10)
+
+// Above Swift 5.10, SwiftPM requires their own AbsolutePath
+// So we have to convert them to Scipio requires
+
+typealias ScipioAbsolutePath = TSCBasic.AbsolutePath
+typealias SwiftPMAbsolutePath = Basics.AbsolutePath
+
+#else
+
+// Below Swift 5.9, Basics.AbsolutePath is not implemented yet. So this is required to keep backward-compatibility
+
+typealias ScipioAbsolutePath = TSCBasic.AbsolutePath
```

Comment:
Can you move it to out side of `#if` block? like below. And please add comment to actual place where we need to `typealias`

```swift
typealias ScipioAbsolutePath = TSCBasic.AbsolutePath

#if swift(>=5.10)
// Above Swift 5.10, SwiftPM requires their own AbsolutePath,
// so we have to bridge them to Scipio requires by typealias
typealias SwiftPMAbsolutePath = Basics.AbsolutePath
#else 
...
```

Reply:
It's easy to understand to define this in both paths